lib/data/item.dart

```diff
@@ -2,37 +2,67 @@ import 'package:cloud_firestore/cloud_firestore.dart';
 
 // In the future we will want to override this class to lost vs. found objects
 class Item {
-  final String? itemName;
-  final String? description;
-  final num? phone;
-  final String? picture;
-  final GeoPoint? location;
+  final String id;
+  final String userId;
+  final Timestamp? timeCreated;
+  final String itemName;
+  final String description;
+  final num phone;
```

Comment:
We should get rid of phone? Because I'm going to update the submission form to not have a phone entry field

Reply:
Yea sure let's get rid of it. That way it forces people to communicate thru our app.